presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -0,0 +1,157 @@
+===============================
+Presto C++ Properties Reference
+===============================
+
+This section describes Presto C++ configuration properties.
+
+The following is not a complete list of all configuration and
+session properties, and does not include any connector-specific
+catalog configuration properties. For information on catalog 
+configuration properties, see :doc:`Connectors </connector/>`.
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Coordinator Properties
+----------------------
+
+Set the following configuration properties for the Presto coordinator exactly 
+as they are shown in this code block to enable the Presto coordinator's use of 
+Presto C++ workers. 
+
+.. code-block:: none
+
+    native-execution-enabled=true
+    optimizer.optimize-hash-generation=false
+    regex-library=RE2J
+    use-alternative-function-signatures=true
+    experimental.table-writer-merge-operator-enabled=false
+
+These Presto coordinator configuration properties are described here, in 
+alphabetical order. 
+
+``experimental.table-writer-merge-operator-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+  Merge TableWriter output before sending to TableFinishOperator. This property must be set to 
+  ``false`` when used. 
```

Comment:
`when used` to me implies that setting this is optional. But this is one of the properties that must be set since the default is not supported by Prestissimo. Is my interpretation wrong?

Reply:
Bit thrown off by this since this seems supported by Prestissimo but likely not PrestoOnSpark https://github.com/prestodb/presto/blob/master/presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestWriter.java#L402

Reply:
We should probably add a test to confirm.

Reply:
removed "when used".